sites/fast-website/src/app/components/navigation/navigation.styles.ts

```diff
@@ -0,0 +1,10 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "@microsoft/fast-components";
+
+export const NavigationStyles = css`
+    ${display("flex")} :host {
+        font-family: var(--body-font);
+        color: var(--neutral-foreground-rest);
```

Comment:
You'll want to make sure you include the `neutralForegroundRestBehavior` as a dependency of this stylesheet

Reply:
thanks, added